libs/sdk-ui/src/base/tests/testHelper.tsx

```diff
@@ -0,0 +1,10 @@
+// (C) 2022 GoodData Corporation
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+export const setupComponent = (jsx: any) => {
```

Comment:
Isn't there some type that can satisfy it?

Reply:
Added it.